Tools/LyTestTools/ly_test_tools/_internal/pytest_plugin/test_tools_fixtures.py

```diff
@@ -67,6 +67,8 @@ def _get_build_directory(config):
         logger.debug(f'Custom build directory set via cli arg to: {custom_build_directory}')
         if not os.path.exists(custom_build_directory):
             raise ValueError(f'Pytest argument "--build-directory" does not exist at: {custom_build_directory}')
+        if custom_build_directory.endswith('debug'):
+            pytest.exit("Python debug modules are not available. LyTestTools test skipped.", 0)
```

Comment:
This will cause pytest to immediately exit with an error code of 2 which signals a failure, which will result in the debug pipeline being red.

Recommend writing a log message (warning?) and then using `sys.exit(0)`

Reply:
I'm passing in an error code of 0, but can switch to sys.exit() if that's preferable

Reply:
Testing this locally, it still returns 2 from pytest. Appears to ignore the code you provide it. Apparently the same goes for sys.exit(0). It appears that PyTest is catching SystemExit exceptions, so it appears another workaround will be needed.

Reply:
Nevermind, if called inside a pytest plugin like this it functions correctly (though apparently sys.exit still misbehaves). My local testing was performed inside an individual test.